officedevtools/officedevtools.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>officedevtools</id>
+    <title>Microsoft Office Developer Tools Preview 2 for Visual Studio 2015</title>
+    <version>14.0.23930</version>
+    <authors>Microsoft</authors>
+    <owners>IOZ AG</owners>
+    <projectUrl>https://www.microsoft.com/en-us/download/details.aspx?id=51683</projectUrl>
+    <packageSourceUrl>https://www.microsoft.com/en-us/download/details.aspx?id=51683</packageSourceUrl>
```

Comment:
Should be ttps://github.com/iozag/chocolatey-packages/tree/master/officedevtools